prod-aws/kafka-shared-msk/data-infra/eqdb-loader.tf

```diff
@@ -1,42 +0,0 @@
-resource "kafka_topic" "di_eqdbloader_schedule" {
```

Comment:
I'll need to manually apply this deletion in prod, so the tf module will error a bit until I do that after merge.